src/docs/advanced/use-sealer-image-offline.md

```diff
@@ -0,0 +1,496 @@
+# Use Sealer Image Offline
+
+## Motivations
+
+It's common that some k8s clusters have their own private image registry, and they don't want to pull images from other registry for some reasons. This page is about how to integrate kyverno into k8s cluster, which will redirect image pull request to Specified registry.
+
+## How to use Kyverno BaseImage
+
+We provide official BaseImage which integrates kyverno into cluster.
+
+| Sealer version |                        Image                         | Arch  |                              OS                              | Network plugins |     Container runtime     |
+| :------------: | :--------------------------------------------------: | :---: | :----------------------------------------------------------: | :-------------: | :-----------------------: |
+|     v0.9.1     |    docker.io/sealerio/kubernetes-kyverno:v1.18.3     |  x86  | CentOS/RHEL 7.5<br/>CentOS/RHEL 7.6<br/>CentOS/RHEL 7.7<br/>CentOS/RHEL 7.8<br/>CentOS/RHEL 7.9 |     calico      | Official docker v19.03.15 |
```

Comment:
modified to this？
docker.io/sealerio/kubernetes-kyverno:v1-18-3-sealerio-1

The following modifications are the same as above. cc@Starnop

Reply:
aggre with you

Reply:
Fixed